docs/content/tutorial/writing-your-first-asset.mdx

```diff
@@ -106,12 +104,12 @@ To follow the progress of the materialization and monitor the logs, each run has
 - Click the value in the **Run ID** column on the table of the Runs page
 - The top section displays the progress, and the bottom section live updates with the logs
```

Comment:
🔧 These guys are still bullets, but should be numbers

Reply:
oh darn it, I enumerated the wrong list earlier!